guides/doc-Upgrading_Project/topics/con_upgrading-prerequisites.adoc

```diff
@@ -19,6 +19,27 @@ For more information, see {InstallingServerDocURL}Preparing_your_Environment_for
 * Back up your {ProjectServer} and all {SmartProxyServers}.
 For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
 * Plan for updating any scripts you use that contain {Project} API commands because some API commands differ between versions of {Project}.
+* If you use an external database, perform the following steps on the {Project} server:
+. Install the `postgresql-contrib` package
```

Comment:
@evgeni Is the package installation required for upgrade as well?

Reply:
It is.

I had opened https://github.com/theforeman/foreman-documentation/pull/2575 for this yesterday.

No idea whether placing it as a prerequisite or inside the main procedure is better.

Two things about your change:
- it needs to happen for 3.6 and 3.7 only (afterwards we can assume it done, as otherwise the system would not be functional)
- the "connect to db" step is different if one doesn't already have a psql shell open (see my PR)

Reply:
Thanks @evgeni ! Looks like I missed that you already pushed the changes for the upgrade guide too. I was confused about whether this should go into prerequisites or the main procedure too. But, I feel it works better your way. I'll close this PR.

Reply:
Agreed @mjivraja closing this one.